Automattic/woocommerce-payments
# Update status chip to WooCommerce pill component for transaction status

Fixes #6060 

#### Changes proposed in this Pull Request









This PR updates the component used to display the status of blocked and held for review transactions. Previously the client used the `TransactionStatusChip` component which was based off the `Chip` component. With this PR, the client now uses the new `TransactionStatusPill` which is based off the `Pill` component. 

As a result of the component change, this PR also updates the relevant unit test snapshots and removes the `TransactionStatusChip` component related files. 

Below are screenshots reflecting the component change.

Screenshot of the blocked transactions using the `TransactionStatusChip` component:

![pr8151-blocked-status-chip](https://github.com/Automattic/woocommerce-payments/assets/60988591/7b22b03a-1b55-43a0-806d-6ba722467c5e)

Screenshot of the blocked transactions using the new `TransactionStatusPill` component:

![pr8151-blocked-status-pill](https://github.com/Automattic/woocommerce-payments/assets/60988591/53e2f270-97e6-43cd-b3a8-e2bb58bd42f3)


#### Testing instructions





* Checkout this PR branch
* Tests should be passing
* Run `npm run build:client`
* Login to your client admin and navigate to Payments > Transactions
* Click the Blocked tab.
* Observe the Status column and ensure the component displaying 'Payment blocked' matches the second screenshot above. Note: you may need to hard refresh your page to pull in the new changes.
* If you don't have any blocked payments:
  * Navigate to Payments > Settings
  * Scroll down to Fraud Protection, select Advanced, then click configure. 
  * Configure the Order Items Threshold rule to set a max of 2 items. 
  * Save settings
  * Place an order with 3 items using test card `4242424242424242`. The order should be blocked.
  * Navigate back to the blocked transactions screen to observe the status column.

**Mobile testing**

* Tube your site: `npm run tube:start`
* Login to the admin via your smartphone
* Navigate to Payments > Transactions
* Click the Blocked tab.
* Observe the Status column and ensure the component displaying 'Payment blocked' matches the second screenshot above.

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : [Update blocked payment status component - PR 8151](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-7.3.0#update-blocked-payment-status-component---pr-8151)
